public/video-ui/src/components/VideoEdit/formComponents/VideoExpiry.js

```diff
@@ -1,27 +1,129 @@
 import React from 'react';
-import DatePicker from '../../FormFields/DatePicker';
+import Picker from 'react-datepicker';
+import moment from 'moment';
+import _ from 'lodash';
+import 'react-datepicker/dist/react-datepicker.css';
+import Icon from '../../Icon';
 
-export default class VideoExpiryEdit extends React.Component {
+function Selector({values, value, disabled, onChange}) {
+  const handler = (e) => {
+    onChange(e.target.value);
+  };
+
+  const options = values.map((value) => {
+    return <option key={value} value={value}>{value}</option>;
+  });
+
+  const params = {
+    className: "form__field form__field--select",
+    disabled: disabled,
+    value: value,
+    onChange: handler
+  };
+
+  return <select {...params}>{options}</select>;
+}
+
+function HourSelector({date, onChange}) {
+  let values = [" "];
+  if (date) {
+    values = [...new Array(24).keys()].map((hour) => moment().hour(hour).format("HH"));
```

Comment:
Related to below, making the picker show up as empty before a date has been selector